homepage/models.py

```diff
@@ -96,3 +97,17 @@ def __str__(self):
         shortened_review = shortened_review[:MAX_LENGTH_PREVIEW]
 
         return f'Preview: {shortened_review}...'
+
+    def print_details(self):
+        message = (
+            f"Course: {self.course}\n"
+            f"User: {self.user}\n"
+            f"Rating: {self.rate}\n"
+            f"{str(self)}\n"    # shortened review content

```

Comment:
```suggestion
            f"Shortened review: {str(self)}\n"    # shortened review content

```

Reply:
I'll change ```def __str__(self)``` last line to ```return f'Shortened review: {shortened_review}...'```
So when calling ```str(self)``` the output will be as expected

Reply:
good idea